src/app/GitUI/LeftPanel/CachedFavorites.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System.Text.Json;
+
+namespace GitUI.LeftPanel
+{
+    public class CachedFavorites
```

Comment:
The members don't have to be marked as internal though.

We generally wish to expose \*only\* required functionality and reduce the public API surface (that can used outside a given assembly) because some of those can _currently_ be used in various plugins.